lm-simulator/lm_simulator/models.py

```diff
@@ -17,6 +17,7 @@ class License(Base):
         lazy="selectin",
         cascade="all, delete-orphan",
     )
+    license_server_type: Mapped[str] = mapped_column(String, nullable=False)
```

Comment:
The License Manager Simulator doesn't use alembic to manage migrations.
It was using a sqlite database until recently where I converted it to use PostgreSQL via docker-compose.
Since it's a project run only locally, I don't know if it's worth adding migrations for now.